tools/api-tests/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  api-test:
+    build: '.'
+    networks:
+      - 'elife-xpub_default'
+    environment:
+      JWT_SECRET:
+
+networks:
+  elife-xpub_default:
+    external:
+      name: 'elife-xpub_default'
```

Comment:
best to keep everything separate at the moment but I would expect that long-term this services to get into https://github.com/elifesciences/elife-xpub/blob/develop/docker-compose.ci.yml